src/mscorlib/src/System/Threading/WaitHandle.cs

```diff
@@ -448,7 +448,7 @@ public static int WaitAny(WaitHandle[] waitHandles, int millisecondsTimeout)
                                         WaitHandle toWaitOn)
         {
 #if PLATFORM_UNIX
-            throw new PlatformNotSupportedException();
+            throw new PlatformNotSupportedException(SR.Arg_PlatformNotSupported); // https://github.com/dotnet/coreclr/issues/10441
```

Comment:
The issue is the specific issue for each one, in this case in the comment. If we don't do the implementation, potentially we should go add a nice message. I don't see this as a particularly high priority vs. adding implementation elsewhere. (If I get a message "Frob is not supported on this platform" vs. "Operation is not supported on this platform." when I call Frob() I don't know I'm greatly better off)

I'm more concerned with the 500-1000 PNSE in CoreFX (note grep doesn't find them all -- there is also `<GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>`). We did go through them once at least.